pytest_mozwebqa/selenium_client.py

```diff
@@ -139,6 +142,16 @@ def start_client(self):
             capabilities.update(webdriver.DesiredCapabilities.OPERA)
             self.selenium = webdriver.Opera(executable_path=self.opera_path,
                                             desired_capabilities=capabilities)
+        elif self.driver.upper() == 'BROWSERSTACK':
```

Comment:
I guess I'm just used to seeing most python code as lowercase, so something entirely uppercase jumps out at me. It's probably not "officially" un-pythonic, but it looks out of place to me. The only things I've ever seen in Python that are uppercase intentionally are constants that are defined at the top of a class. I think it's fine - just something I wanted to point out/ask about.